src/tools/copyset_check_core.cpp

```diff
@@ -476,9 +477,36 @@ int CopysetCheckCore::CheckCopysetsWithMds() {
         std::cout << std::endl;
         ret = -1;
     }
+
+    // Check scan status for inconsistent copyset
+    auto nInconsistent = CheckScanStatus(copysetsInMds);
+    if (nInconsistent > 0) {
```

Comment:
Can you print detailed info about which copyset/chunk and which part of data is inconsistent?

Reply:
> Can you print detailed info about which copyset/chunk and which part of data is inconsistent?

Actually, the detail will been print in [CopysetCheck::PrintDetail](https://github.com/opencurve/curve/blob/5ae2618538285503c04aaf287f6a2a0480acd68f/src/tools/copyset_check.cpp#L258) if you add the detail in `copysets_` map (I has added the detail into it)